build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessLogcatTiming.cs

```diff
@@ -16,19 +16,47 @@ public class ProcessLogcatTiming : Task
 		[Required]
 		public string ApplicationPackageName { get; set; }
 
+		[Required]
+		public string DefinitionsFilename { get; set; }
+
 		public string ResultsFilename { get; set; }
 
+		Dictionary<string, Regex> definedRegexs = new Dictionary<string, Regex> ();
+		Dictionary<string, string> results = new Dictionary<string, string> ();
+
+		void LoadDefinitions ()
+		{
+			using (var reader = new StreamReader (DefinitionsFilename)) {
+				string line;
+
+				while ((line = reader.ReadLine ()) != null) {
+					int index = line.IndexOf (';');
+					if (index < 1 || index == line.Length)
+						continue;
+					var label = line.Substring (0, index);
+					var pattern = line.Substring (index + 1);
+					Regex regex;
+					try {
+						regex = new Regex (pattern);
+					} catch (Exception e) {
+						Log.LogWarning ($"unable to create regex for label: {label} from pattern: {pattern}\n{e}");
+						continue;
+					}
+					definedRegexs [line.Substring (0, index)] = regex;
```

Comment:
Additionally, is this wise or desirable? This allows overwriting previous labels, e.g. with a file of:

    a;some-regex
    a;another-regex
    a;do we have enough duplicates yet?

Is it really a good idea for the "last" one to replace anything previously defined, without even a warning?